### ISSUE TITLE
Modals not working if HTMX Navigation is enabled

### ISSUE THREAD

**Issue by arthanson** [netbox-community/netbox#15580]
### Deployment Type

Self-hosted

### NetBox Version

v4.0.0-dev

### Python Version

3.8

### Steps to Reproduce

1. Go to user preferences and set "HTMX Navigation" to Enabled
2. Go to Devices and click the "Configure Table" button

### Expected Behavior

The configure table modal should pop up

### Observed Behavior

No modal appears.

**Comment by jeremystretch** [comment]
AFAICT this occurs only after certain actions, such as visiting the home page or updating preferences. Navigating directly to an object list view and then navigating normally via the menu with HTMX enabled works fine; the table config dialogs open as expected.

**Comment by opericgithub** [comment]
Config table window opens now, but I'm unable to move columns from one list to another (add and remove buttons don't function).

**Comment by jeremystretch** [comment]
@opericgithub please open a new issue.
